.github/workflows/build-docs.yml

```
@@ -16,7 +16,7 @@ jobs:
   build-gmp-doc:
     name: Build GMP documentation
     runs-on: ubuntu-latest
-    container: greenbone/gvmd-build:stable
+    container: gchr.io/greenbone/gvmd-build:stable
```

```suggestion
    container: ${{ vars.IMAGE_REGISTRY }}/greenbone/gvmd-build:stable
```

(not sure if it also works here ...)